.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /Cargo.lock
```

Comment:
Curious why are we ignoring the lock file ?

Reply:
IMHO we should not be. I'm going to commit a current one and ensure that we use it for CI.